lib/main_micro_alp.pm

```diff
@@ -204,8 +204,8 @@ sub load_network_tests {
 sub load_qemu_tests {
     loadtest 'microos/rebuild_initrd' if is_s390x;
     loadtest 'qemu/info';
-    loadtest 'qemu/qemu';
-    loadtest 'qemu/kvm' unless is_aarch64;
+    loadtest 'qemu/qemu' unless is_rt;
+    loadtest 'qemu/kvm' unless (is_aarch64 or is_rt);
```

Comment:
why not in aarch64? It seems to work fine.

Reply:
I did not do anything to arm.  ```unless is aarch64``` already there.

Reply:
It comes from here this file: schedule/sle-micro/virt.yaml

Reply:
I double checked corresponding test in SLE Micro 5.4, there is no kvm test for aarch64 either, please refer to https://openqa.suse.de/tests/10855751

Reply:
According to susedoc, it seems nested kvm is only supported with intel or amd cpu, please refer to https://susedoc.github.io/doc-sle/main/single-html/SLES-virtualization/#sec-vt-installation-nested-vms. There is no relevant description about how to enable this with aarch64.

Reply:
I think that nested virtualization is not supported on our arm workers that we have. This might have change by now though. Regarding RT, well the pattern is not preinstalled by default. I do not remember if it should work or not or if we should support it. In that case, it does not make much sense to run that test suite against the image.

Reply:
Generally, I would say running VMs on top of RT kernel does not seem to me like a valid user case. Or at least it is very peculiar IMHO

Reply:
> Generally, I would say running VMs on top of RT kernel does not seem to me like a valid user case. Or at least it is very peculiar IMHO

Yes,I confirmed this. RT is not supporting virtualization.

Reply:
> I did not do anything to arm. `unless is aarch64` already there. But aarch64 does not support nested kvm might be the reason, so there is no need to test kvm in qemu vm ? The possible reason does not originate from me, I just try my best to recall something that had ever crossed my brain.

ok, sorry, I miss-read it. Yes, it's ok then.

Reply:
@mloviska Anything else ? Or can be merged ?

Reply:
@jlausuch  Hi Jose, with clarification that RT is not supporting virtualization, JFYI, we will remove such test on RT in virtualization job group -- https://openqa.suse.de/tests/11768423.

Reply:
BTW, it seems to me this PR can be merged. Will do it if no further reply before EOB today.